packages/components/src/number-control/index.tsx

```diff
@@ -258,9 +256,7 @@ function UnforwardedNumberControl(
 									className={ spinButtonClasses }
 									icon={ resetIcon }
 									size="small"
-									aria-hidden="true"
-									aria-label={ __( 'Decrement' ) }
```

Comment:
Why would someone put an `aria-label` on something that is inaccessible to keyboard users? Makes no sense, sigh.